app/controllers/categories_controller.rb

```diff
@@ -5,5 +5,6 @@ def index
 
   def show
     @category = Category.find(params[:id])
+    @staches = @category.staches
```

Comment:
Could we perhaps just use the one instance variable and access the staches that way? Maybe use #includes to do some happy awesome eager loading?